packages/wonder-blocks-popover/src/components/__tests__/popover-dialog.test.js

```diff
@@ -1,45 +1,27 @@
 // @flow
 import * as React from "react";
-import {mount} from "enzyme";
-import "jest-enzyme";
-
-import {TooltipTail} from "@khanacademy/wonder-blocks-tooltip";
+import {render} from "@testing-library/react";
 
+import type {Placement} from "@khanacademy/wonder-blocks-tooltip";
 import PopoverDialog from "../popover-dialog.js";
 import PopoverContentCore from "../popover-content-core.js";
 
 jest.mock("@khanacademy/wonder-blocks-tooltip");
 
 describe("PopoverDialog", () => {
-    it("should update the tail color to match the content's color", () => {
-        // Arrange
-        const onUpdateMock = jest.fn();
-
-        // Act
-        const wrapper = mount(
-            <PopoverDialog placement="top" onUpdate={onUpdateMock}>
-                <PopoverContentCore color="darkBlue">
-                    popover content
-                </PopoverContentCore>
-            </PopoverDialog>,
-        );
-
-        // Assert
-        expect(wrapper.find(TooltipTail).prop("color")).toBe("darkBlue");
-    });
```

Comment:
**question:** Is there another way we can validate this? It seems like we want to make sure it's still the case somehow. One option is to just mock the tail import and capture the props, for example. There may be some other way such as looking for specific HTML in the `container.innerHTML` returned by `render`.